components/navbar.css

```diff
@@ -0,0 +1,34 @@
+.nav-container {
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	background-color: var(--primary-color);
+	font-size: 2.4rem;
+	padding: 1.6rem 0;
+	margin: 0 1.6rem;
+	box-shadow: 2px 2px 8px var(--primary-color);
```

Comment:
if you are using this `box-shadow` color multiple times, you may store it in a variable.